src/swupd.h

```diff
@@ -42,6 +42,7 @@ extern "C" {
 #define MIX_CERT MIX_DIR "Swupd_Root.pem"
 #define MIX_BUNDLES_DIR MIX_STATE_DIR "mix-bundles/"
 #define MIXED_FILE SWUPD_DEFAULTS "mixed"
+#define SSL_CLEINT_CERT "/etc/swupd/client.crt"
```

Comment:
I'm not sure if we want this in /etc?  I could see this being something we want as a part of the mixing process that could perhaps live in /usr/share/clear somewhere.

Reply:
I think it would make sense to have an override given we can point to a new mirror.  The new mirror could conceivably need a different SSL client cert.

Reply:
I'm not totally against the cert in /etc but...

> The new mirror could conceivably need a different SSL client cert.

I hope this is never the case.